recipes/salmon/salmon_cmake.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -45,7 +45,7 @@ if(CONDA_BUILD)
+   set(CMAKE_CXX_STANDARD_REQUIRED ON)
+   set(CMAKE_CXX_EXTENSIONS OFF)
+   set(CXXSTDFLAG "-std=c++11")
+-  set(GCCVERSION "4.8.5")
++  set(GCCVERSION "4.8.2")
```

Comment:
Good to have upstream developer support :)
Thanks @rob-p! Seems to work for Linux now. If OSX passes I will clean up the recipe a little bit and we can merge it.

Reply:
Great!  I'm not sure if OSX will work or not yet though --- there was a reason we had all of that `llvm_dev` jazz there before ;P.

Reply:
Ok, so i think the osx error is due to the compiler version.  Is there anything we can do at this point to get llvm_dev, or do we have to wait for something regarding cb3?